.github/workflows/build-back-end-development.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     if: github.event.pull_request.merged
 
     steps:
+      - name: Checkout
```

Comment:
Why was the checkout needed here? 🤔 
Would this parameter be the one your looking for https://github.com/ams0/acr-task-github-action/blob/87d2021ccffb1ffb462b5bc70289c7dcdf260bcb/action.yml#L29

Reply:
it seems to default on master, so that might be the problem your trying to tackle here

Reply:
Ok, didn't check the acr's documentation because I assumed (root of all evil) that the task wouldn't change the build context middle of the steps.